src/StoreKeeper/WooCommerce/B2C/Models/TaskModel.php

```diff
@@ -135,15 +136,15 @@ public static function getLastProcessTaskDate()
 
         $task = reset($results);
 
-        return reset($task);
+        return DatabaseConnection::formatFromDatabaseDate(reset($task));
```

Comment:
```suggestion
        return DatabaseConnection::formatFromDatabaseDateIfNotEmpty(reset($task));
```

also change in other places

Reply:
Changed for necessary codes